src/CMakeLists.txt

```diff
@@ -603,13 +603,25 @@ endif()
 
 ############################################################
 # Installation
+if(MIOPEN_INSTALL_CXX_HEADERS)
 rocm_install_targets(
   TARGETS MIOpen
   INCLUDE
     ${PROJECT_SOURCE_DIR}/include
     ${PROJECT_BINARY_DIR}/include
+    ${PROJECT_SOURCE_DIR}/src/include
```

Comment:
Can you use a variable for this instead of duplicating the `rocm_install_targets` command?

```cmake
set(MIOPEN_CXX_HEADER_PATH)
if(MIOPEN_INSTALL_CXX_HEADERS)
set(MIOPEN_CXX_HEADER_PATH ${PROJECT_SOURCE_DIR}/src/include)
endif()

rocm_install_targets(
  TARGETS MIOpen
  INCLUDE
    ${PROJECT_SOURCE_DIR}/include
    ${PROJECT_BINARY_DIR}/include
    ${MIOPEN_CXX_HEADER_PATH}
  PREFIX ${MIOPEN_INSTALL_DIR}
)
```

Reply:
Fixed